doc/releases/changelog-dev.md

```diff
@@ -20,24 +20,36 @@ keyword argument when using `GellMann`, which determines which of the 8 Gell-Man
     >>> generators = qchem.symmetry_generators(H)
     >>> paulixops = qchem.paulix_ops(generators, n_qubits)
     >>> paulix_sector = qchem.optimal_sector(H, generators, mol.n_electrons)
-    >>> qchem.taper_operation(qml.SingleExcitation(3.14159, wires=[0, 2]),
-                                generators, paulixops, paulix_sector, wire_order=H.wires)
-    [PauliRot(-3.14159+0.j, 'RY', wires=[0])]
-    ```
+    >>> qchem.taper_operation(qml.SingleExcitation(3.14159, wires=[0, 2]), generators, paulixops,
+                        paulix_sector, wire_order=H.wires, op_wires=[0, 2])
+    [Exp(1.570795j, 'PauliY', wires=[0])]
+  ```
+
+  This can also be used with the functional form of the operation:
+
+  ```pycon
+    >>> tap_op = qchem.taper_operation(qml.SingleExcitation, generators, paulixops, 
+                    paulix_sector, wire_order=H.wires, op_wires=[0, 2])
+    >>> tap_op(3.14159)
+    [Exp(1.570795j, 'PauliY', wires=[0])]
+  ```
```

Comment:
@obliviateandsurrender is the functional form preferred due to performance improvements?

If so, I would alter the docstring to only show the functional form in the minimal example, and move the OOP form to Usage Details.

You can think of the docstring as the 'convention setter' for users. The first example they see will likely set the convention that they use :)